Directory.Build.props

```diff
@@ -8,12 +8,6 @@
     <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory).sonarlint\sonarlint-visualstudioCsharp.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
-  <ItemGroup Label="Common items">
```

Comment:
This is not actually related to the PR, it was a way for me to test that the fix works. I can revert it and push it in another commit (that will also have the server project binding changes).

Reply:
Personally, I'd probably revert it since it isn't part of the fix, and the only thing that will change when we fix up the project binding is the path in line 12. However, it doesn't matter much either way.